problems/1.types+operators.js

```diff
@@ -2,16 +2,16 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+const isNumber = (val) => {
+  return typeof val === 'number'
 }
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
-
+const isTruthy = (val) => {
+  return typeof val !== false
```

Comment:
typeof evaluates to a string that says the type. We want to check for truthyness. Try using a conditional or !!.